sounds/models.py

```diff
@@ -2433,7 +2430,7 @@ def get_analysis_data_from_file(self):
             pass
         try:
             with open(self.analysis_filepath_base + ".yaml") as f:
-                return yaml.load(f, Loader=yaml.cyaml.CSafeLoader)
+                return yaml.safe_load(f, Loader=yaml.cyaml.CSafeLoader)
```

Comment:
I did this because the linter said to, but haven't checked if it's actually valid - I guess our yaml files are data only, so we shouldn't have any problem switching between load or safe load. Note that we use load with a specific safe loader anyway, so is this actually an issue??

Reply:
yaml was historically very insecure, and there are many workarounds to make it safer. We can review these arguments in more detail to determine the best fix, I'll do this